frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx

```diff
@@ -40,12 +41,26 @@ export const scene: SceneExport = {
 }
 
 export function EarlyAccessFeature({ id }: { id?: string } = {}): JSX.Element {
-    const { earlyAccessFeature, earlyAccessFeatureLoading, isEarlyAccessFeatureSubmitting, isEditingFeature } =
-        useValues(earlyAccessFeatureLogic)
+    const {
+        earlyAccessFeature,
+        earlyAccessFeatureLoading,
+        isEarlyAccessFeatureSubmitting,
+        isEditingFeature,
+        earlyAccessFeatureMissing,
+    } = useValues(earlyAccessFeatureLogic)
     const { submitEarlyAccessFeatureRequest, cancel, editFeature, updateStage, deleteEarlyAccessFeature } =
         useActions(earlyAccessFeatureLogic)
 
     const isNewEarlyAccessFeature = id === 'new' || id === undefined
+
+    if (earlyAccessFeatureMissing) {
+        return <NotFound object="early access feature" />
+    }
+
+    if (earlyAccessFeatureLoading) {
+        return <LemonSkeleton active />
```

Comment:
I think a <Spinner /> is better than a single random skeleton... Skeletons should be used to build a mock UI in a loading state. If that's too much effort, spinner is the way to go I'd say.

Reply:
oh, interesting. For things like this one which is just querying postgres, and is relatively fast in the happy path, I find the spinner to be a bit more jarring, since it just flickers in the middle of the screen and goes away, whereas the skeleton is a bit more seamless.

How do you usually handle cases like these?